XpringKit/JavaScript/JavaScriptSerializer.swift

```diff
@@ -31,11 +36,11 @@ internal class JavaScriptSerializer {
     return walletClass.construct(withArguments: [ wallet.publicKey, wallet.privateKey])
   }
 
-  /// Serialize a transaction to a JavaScript object.
+  /// Serialize a legacy transaction to a JavaScript object.
```

Comment:
Should this have legacy in the comment, since this serialize takes an `Rpc_V1_Transaction`?

Reply:
Done. Botched copy / paste job on my part.